assets/js/controllers/signup.controller.js

```diff
@@ -153,6 +153,8 @@ function SignupCtrl ($scope, $state, $cookies, $filter, $timeout, $translate, Wa
     email: $state.params.email || ''
   };
 
+  $state.params.email ? $scope.fields.emailIsFromState = true : '';
```

Comment:
Someone quickly reading this my see `emailIsFromState == true` instead of `=true` and not realize that this line is assigning a variable. The following would be better:
`$scope.fields.emailIsFromState = !!$state.params.email` or `$scope.fields.emailIsFromState = Boolean($state.params.email)`